lib/praxis/api_general_info.rb

```diff
@@ -40,6 +45,14 @@ def description(val=nil)
       end
     end
 
+    def version(val=nil)
```

Comment:
I think we should name this version_using  (and drop the using: key param ... it does not make sense in here as there is no version param to be passed anyway).
The DSL will also be clearer by saying `using_version :path` ...